src/markus/backends/statsd.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, options=None, filters=None):
         self.client = self._get_client(
             self.host, self.port, self.prefix, self.maxudpsize
         )
-        logger.info(
+        logger.debug(
```

Comment:
I don't know why this was at the info level. It should be debug.

Reply:
I was unable to get this to log in either case, because we setup `markus` in Django settings before logging is setup. Maybe we need to move our logging up...